assets/js/hooks/session.js

```diff
@@ -988,6 +993,25 @@ const Session = {
       }
     }
   },
+  customViewSelection() {
+    settingsStore.getAndSubscribe((settings) => {
+      settings.custom_section
+        ? this.el.setAttribute("data-js-section", true)
+        : this.el.removeAttribute("data-js-section");
```

Comment:
Also, let's make the attribute more clear, e.g. `data-js-show-section`.

Reply:
I propose to be the opposite: "data-js-hide-section" since that's how we'd best use it on the CSS front. If we want to call it show, I would go with "data-js-show-section=false". Thoughts @jonatanklosko?

Reply:
`data-js-hide-section` totally works! My point was just to make it clear that it's a toggle :D